src/components/UnorderedList/index.js

```diff
@@ -0,0 +1,6 @@
+/**
+ * @file Unordered List entry point.
```

Comment:
Small sentence case suggestion (for consistency with other files)
```suggestion
 * @file Unordered list entry point.
```